### PR TITLE
fixes in requirements and readme

### PR DESCRIPTION
**Pull request by timliubentoml**
Updated the readme, imports and requirements based on what was tested on clean ubuntu20

### CODE REVIEW

File: tensorflow2/service.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import bentoml
 import numpy as np
+import PIL.Image
```
Comment (aarnphm) [inline tensorflow2/service.py:3]: Why do we need to import this line @timliubentoml ?

Comment (aarnphm) [reply tensorflow2/service.py:3]: at line 5 `PIL.Image` should already been imported

Comment (timliubentoml) [reply tensorflow2/service.py:3]: Yea, wasn't sure. It wasn't working and the pytorch_mnist had it like this.

Comment (aarnphm) [reply tensorflow2/service.py:3]: line 5 is a bit of a Python hack, so I think this would be good to add.

**Review by aarnphm — APPROVED** [review]
LGTM. Tested locally on my end and everything still works as expected.

**Pull request merged**